### PR TITLE
MAINT: Update cversion hash.

### LINKED ISSUE #12412: MismatchCAPIWarning during build
Something we might want to deal with before 1.16:

> numpy\core\setup.py:400: MismatchCAPIWarning: API mismatch detected, the C API version numbers have to be updated. Current C api version is 13, with checksum 5b0e8bbded00b166125974fc71e80a33, but recorded checksum for C API version 13 in codegen_dir/cversions.txt is a1bc756c5782853ec2e3616cf66869d8. If functions were added in the C API, you have to update C_API_VERSION  in numpy\core\setup_common.py.

The build continues despite this warning, so it's possible we just didn't notice it. This is likely my fault as part of #8955

### PR DESCRIPTION
**Pull request by charris**
A new function was added to the ufunc_funcs_api.

Closes #12412.

### CODE REVIEW

File: numpy/core/code_generators/cversions.txt [hunk 1]
```diff
@@ -43,7 +43,8 @@
...
 0x0000000c = a1bc756c5782853ec2e3616cf66869d8
 
-# Version 13 (NumPy 1.16) Deprecate PyArray_SetNumericOps and
-# PyArray_GetNumericOps, Added fields core_dim_flags and core_dim_sizes
-# to PyUFuncObject
-0x0000000d = a1bc756c5782853ec2e3616cf66869d8
+# Version 13 (NumPy 1.16)
+# Deprecate PyArray_SetNumericOps and PyArray_GetNumericOps,
+# Add fields core_dim_flags and core_dim_sizes to PyUFuncObject.
+# Add pyufunc_fromfuncanddataandsignatureandidentity to ufunc_funcs_api.
```
Comment (eric-wieser) [inline numpy/core/code_generators/cversions.txt]: Should this not have the same case as the function name?

File: numpy/core/code_generators/numpy_api.py [hunk 2]
```diff
@@ -402,8 +402,7 @@
     # End 1.7 API
     'PyUFunc_RegisterLoopForDescr':             (41,),
     # End 1.8 API
-    'PyUFunc_FromFuncAndDataAndSignatureAndIdentity':
-                                                (42,),
+    'PyUFunc_FromFuncAndDataAndSignatureAndIdentity': (42,),
```
Comment (eric-wieser) [inline numpy/core/code_generators/numpy_api.py]: Note that I wrapped this to make the columns align - not that it matter at all

**Comment by eric-wieser** [comment]
Perhaps that warning should be promoted to an error at build time, else we'll never catch it.

**Comment by charris** [comment]
Hah, that was strange. I upgraded to Fedora 28 last Thurs, and among various new vim bugs, it turns out it was lower casing strings copied to the clipboard. Seems vim is not maintained as well as it was...

**Comment by charris** [comment]
> Perhaps that warning should be promoted to an error at build time, 

Sounds like a good idea.

**Review by eric-wieser — APPROVED** [review]

**Pull request merged**